Classes/Service/ConfigurationService.php

```diff
@@ -351,4 +365,16 @@ protected function sendDisabledContentWarning($templatePathAndFilename) {
 		$this->message('Disabled Fluid Content Element: ' . $templatePathAndFilename, GeneralUtility::SYSLOG_SEVERITY_NOTICE);
 	}
 
+	/**
+	 * @param string $sourceFile
+	 * @param integer $maxWidth
+	 * @param integer $maxHeight
+	 * @return string
+	 */
+	protected function resizeImage($sourceFile, $maxWidth, $maxHeight) {
+		$cmd = GeneralUtility::imageMagickCommand('convert', "+profile '*' -geometry " . $maxWidth . 'x' . $maxHeight . "!   '" . $sourceFile . "' '/var/www/typo3temp/pics/" . sha1($sourceFile) . ".png'");
+		CommandUtility::exec($cmd);
+		return '/typo3temp/pics/' . sha1($sourceFile) . '.png';
```

Comment:
Please use a utility and maybe the upload folder. temp is often cleared and wizard is cached